modules/virt-schedule-cpu-host-model-vms.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-schedule-vms.adoc
+
+[id="virt-schedule-cpu-host-model-vms_{context}"]
+= Scheduling virtual machines with the host model
+
+When the CPU model for a virtual machine (VM) is set to `host-model`, the VM inherits the CPU model of the node where it is scheduled.
+
+.Procedure
+
+* Edit the `domain` spec of your VM configuration file. The following example shows `host-model` being specified for the Virtual Machine Instance (VMI):
```

Comment:
In other modules, you used lowercase "virtual machine instance (VMI)". Can you update this instance to be consistent?

Reply:
Updated it to lower case. Squashed the commits. Thanks!